.github/workflows/windows.yml

```diff
@@ -19,9 +44,15 @@ jobs:
       - name: setup
         run: |
           pip3 install pytest tox
-
-      - name: compile
+          mkdir builds
+      - name: download-planner
+        uses: actions/download-artifact@master
+        with:
+          # without 'name' attribute all artifacts are downloaded and the
+          # artifact name is used as directory name.
+          path: builds/
+      - name: test
         shell: cmd
         run: |
-          "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Enterprise\\VC\\Auxiliary\\Build\\vcvarsall.bat" x64 & python build.py
-          "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Enterprise\\VC\\Auxiliary\\Build\\vcvarsall.bat" x64 & python build.py --debug
+          cd misc/
+          tox -e search
```

Comment:
It's great to see that this works! I didn't know this. So tox somehow is able to correctly run all scripts on Windows... :)